guide/07-working-with-bigdata/02-making-your-data-accessible-to-ga-server.ipynb

```diff
@@ -0,0 +1,838 @@
+{
```

Comment:
<p>In this guide, we have covered concepts on registering datastores and big data file shares, inspecting a big data file share, modifying data schema, and more. In this next guide, we will perform analysis with the published data and learn more about tools available in the geoanalytics module.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1076/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>